src/components/DocumentImageSection.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled from 'styled-components/native'
+
+import { Document } from '../constants/endpoints'
+import AudioPlayer from './AudioPlayer'
+import FavoriteButton from './FavoriteButton'
+import ImageCarousel from './ImageCarousel'
+
+const AudioContainer = styled.View`
+  position: absolute;
+  bottom: ${wp('-4.5%')}px;
+  align-self: center;
+`
+const FavoriteContainer = styled.View`
+  position: absolute;
+  top: 0;
+  right: ${props => props.theme.spacings.md};
+`
+
+const Container = styled.View`
+  margin-bottom: ${props => props.theme.spacings.md};
+`
+
+interface Props {
+  document: Document
+  audioDisabled?: boolean
+  minimized?: boolean
+  submittedAlternative?: string | null
+}
+
+const DocumentImageSection = ({
```

Comment:
(opional) rename to WordImageSection

Reply:
Why? Did I miss that we decided to switch from document to word naming wise? afaik it is always called document

Reply:
Yeah, naming is messy, thats why I marked it opional. But I think we will replace document naming, as these are not really documents. :thinking:

Reply:
Hmm word is not really a good alternative though imo, it should probably be called something like vocab or similar. But since right now document is the current standard I'll keep it, may be renamed when we decide to switch to something else.

Reply:
fine for me. But vocab is - like vocabulary - only a plural word, so not really fitting.

Reply:
yeah, we can discuss this when we want to change this :)